auto_trainer/auto_trainer.py

```diff
@@ -119,6 +119,7 @@ def train(
     test_set: DataItem = None,
     train_test_split_size: float = None,
     random_state: int = None,
+    labels: Dict[str, Union[str, int, float]] = None,
```

Comment:
```suggestion
    labels: dict = None,
```

I think its obvious what is a label dictionary :)